spec/controllers/invite_requests_controller_spec.rb

```diff
@@ -60,11 +60,24 @@
       expect(response).to render_template("index")
     end
 
-    it "redirects to index with error given valid emails" do
-      email = generate(:email)
-      post :create, params: { invite_request: { email: email } }
-      invite_request = InviteRequest.find_by_email(email)
-      it_redirects_to_with_notice(invite_requests_path, "You've been added to our queue! Yay! We estimate that you'll receive an invitation around #{invite_request.proposed_fill_date}. We strongly recommend that you add do-not-reply@archiveofourown.org to your address book to prevent the invitation email from getting blocked as spam by your email provider.")
+    context "with valid emails" do
+      let(:ip) { "867.53.09" }
+
+      before do
+        allow_any_instance_of(ActionDispatch::Request).to receive(:remote_ip).and_return(ip)
+      end
+
+      it "redirects to index with error given valid emails" do
+        email = generate(:email)
+        post :create, params: { invite_request: { email: email } }
+        invite_request = InviteRequest.find_by_email(email)
```

Comment:
This is meant to test if the invite request is actually created, so we should use `find_by!` to fail the test if the invite is not found.